content/post/career-ladders-part-2.md

```diff
@@ -0,0 +1,56 @@
+---
+author: Caitlin Moorman
+title: "How to Create a Career Ladder"
+cover: ""
+draft: true
+publishDate: "2019-01-13"
+tags:
+ - analytics
+ - org
+ - teams
+ - management
+---
+
+After last week, I'm sure you're bought in on [why a career ladder is so important](https://www.locallyoptimistic.com/post/career-ladders-part-1/), so let's jump right into creating one that works for your team. I separate a good career ladder into two parts: guiding principles and specific competencies.
+
+## Guiding Principles
+Guiding principles are simple heuristics that capture how roles differ from each other. You don’t get promoted for just getting better at the same job, you get promoted for showing that you can do a different job.
```

Comment:
What we like to say is that you have to have been performing at the level above for some time. Not sure if this reinforces some version of being taken advantage of (I don't think it does), but could be worth adding to the end, e.g `...showing that you've already been operating at the next level`